components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounter.kt

```diff
@@ -259,11 +258,11 @@ open class TabCounter @JvmOverloads constructor(
     }
 
     companion object {
+        internal var INTERNAL_COUNT = 0
```

Comment:
Please don't use all cap for variable that is not constant.

Reply:
This was necessary in Fenix because count got reset while navigating. I did it also here too.
https://github.com/mozilla-mobile/fenix/pull/11738